zerver/webhooks/dialogflow/doc.md

```diff
@@ -0,0 +1,24 @@
+Get personal message notifications in Zulip for the results of your Dialogflow queries!
+
+{!create-bot-construct-url.md!}
+
+The webhook URL should look like:
+
+`{{api_url}}?api_key=BOT'S_API_KEY&email=foo@example.com`
+
```

Comment:
Yeah, so you either use this URL or the one generated by `{!create-bot-construct-url.md!}`. You can't have two webhook URLs. Does that make sense?

Reply:
okay! got it.